test/test_jit_fuser_te.py

```diff
@@ -1858,6 +1858,23 @@ def eager(x, y):
         script = self.checkScript(eager, (x, y))
         self.assertAllFused(script.graph_for(x, y))
 
+    def test_channels_last_dims_dynamic(self):
```

Comment:
@navahgar, @ZolotukhinM this runs into an IR Simplifier bug. I tested this on simpler functions with a single input and was able to run successfully but now hitting simplifier bug.